rosbridge_library/test/internal/test_message_conversion.py

```diff
@@ -21,6 +21,10 @@
 
 
 class TestMessageConversion(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        c.configure()
```

Comment:
Should this be `cls.configure()`?

Edit: oh no there is just a module being imported as `c` from before. Also gross and prone to misinterpretation, can we not alias that?

Reply:
I agree the alias makes it less readable. I removed the aliases from the tests